RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
+
+As much as possible, we will try to maintain a `master` branch that is compatible with all supported versions of Kubernetes.
+If there is a variation in the Kubernetes API which requires a _**breaking change**_ to the Addon, a _**branch**_ will be made for the prior Kubernetes versions.
```

Comment:
A similar scenario, when do we add new addons, what about making an addon enabled by default?

When upgrading from Konvoy 1.2 to 1.3 there were multiple new addons that were required, do we need to distinguish between new addons that are enabled by default vs new addons that are required?

Reply:
> do we need to distinguish between new addons that are enabled by default vs new addons that are required?

it might make sense for us to audit all the addons and only leave the "required" ones in this repo (the ones absolutely critical for production ready kubernetes). 

Now that we can add multiple configRepos in cluster.yaml we can move out the non-essential addons to their own repos

Reply:
> it might make sense for us to audit all the addons and only leave the "required" ones in this repo (the ones absolutely critical for production ready kubernetes).

I completely agree.

Reply:
For Dimitri's "similar scenario", I'll update the document.

Reply:
Actually, I've changed my mind on this one. Earlier this morning we began a conversation that would allow konvoy to pick up the newly added addons and prompt the user to enable all, some, or none. This won't be part of this document.